### ISSUE TITLE
TerminalView roadmap

### ISSUE THREAD

**Issue by Wramberg** [Wramberg/TerminalView#49]
Just to let everyone know about my development plans (and to remember everything :-)). PRs are always welcome regardless of the roadmap.

### 0.6.0
* Pyte -> GateOne terminal emulator
  * xterm support
  * 256 color support
* Editor mode where you can essentially turn the TerminalView into a normal view temporarily to use your standard sublime keybindings
* Fix for unicode issue
* Unittests cleanup (need better structure and tests that are easier to write)
* Unittests for zsh and fish

### 0.7.0
* Support displaying images in terminal views

### Unscheduled
* Using ST3 scrolling instead of terminal emulator scrolling
* Dynamic amount of scrolling instead of fixed ratio
* QOL shortcut that can find and open filepaths (depends alot on the program that prints the filepath)
* Windows support (this is going to be hard - did some tests with WSL but no luck)

**Comment by rwols** [comment]
What about a way to convert your color scheme into a "terminal" color scheme? I've been toying around with this but I haven't had much luck so far.

**Comment by Wramberg** [comment]
What do you mean ? Like converting regular color schemes into TerminalView color schcmes or ?

**Comment by rwols** [comment]
Yeah. Take the colors of your color scheme and fit them into the 8 color buckets of a terminal in a sensible way basically.

**Comment by Wramberg** [comment]
That sounds like a complex problem. On one hand you have a shell requesting something be shown in `red`. On the other, you have a color scheme with keys like `keyword.control`. How would you choose the red color ?

**Comment by rwols** [comment]
PR here: https://github.com/Wramberg/TerminalView/pull/53

**Comment by spacekookie** [comment]
Would it be possible to hook the scrolling into the ST3 preview so that the command history can be seen in it?
